python/core/auto_generated/qgis.sip.in

```diff
@@ -303,6 +303,7 @@ The development version
       FlatFace,
       UniqueSharedVertex,
       InvalidVertex,
+      FacesLinkWithSameClockwise,
```

Comment:
not sure about name? Do you mean that all faces have the same orientation? maybe `FacesHasSameOrientation`?

Reply:
yes, better...

Reply:
hmmm, thinking about,  `ManifoldFace` is better, because faces have to have same orientation